js/main.js

```diff
@@ -144,10 +150,21 @@ function onSearch(event) {
   jikanReq.send();
   switchViews('search-results');
   $searchBarResults.value = data.search;
+
+  jikanReq.onerror = function () {
+    $loadBar.className = 'lds-facebook hidden';
+    $networkErrorHeader.className = 'network-error-header text-align-center';
+  };
```

Comment:
You should use `addEventListener` to match the format of your `load` event handler